apps/wallet-mobile/src/kernel/i18n/locales/en-US.json

```diff
@@ -1220,5 +1220,19 @@
   "portfolio.portfolioTokensDetailScreen.6_months": "6 months",
   "portfolio.portfolioTokensDetailScreen.1_year": "1 year",
   "portfolio.portfolioTokensDetailScreen.all_time": "all time",
-  "portfolio.portfolioTokensDetailScreen.detailsOn": "Details on"
+  "portfolio.portfolioTokensDetailScreen.detailsOn": "Details on",
+  "txReview.tabLabel.utxos": "UTxOs",
+  "txReview.tabLabel.overview": "Overview",
+  "txReview.overview.wallet": "Wallet",
+  "txReview.fee": "Fee",
+  "txReview.overview.myWalletLabel": "Your Wallet",
+  "txReview.overview.sendLabel": "Send",
+  "txReview.overview.receiveToLabel": "receiveToLabel",
```

Comment:
`receiveToLabel` is this translation correct?

Reply:
fixed in the next pr @stackchain  👀